lib/graphql/execution/multiplex.rb

```diff
@@ -161,7 +161,7 @@ def with_instrumentation(schema, queries, context:, max_complexity:)
           end
 
           multiplex_analyzers = schema.multiplex_analyzers
-          if max_complexity ||= schema.max_complexity
```

Comment:
This breaks `GraphQL::Analysis::MaxQueryComplexity::across a multiplex#test_0001_returns errors for all queries:`!